src/api/app/views/webui/users/tokens/_actions.html.haml

```diff
@@ -0,0 +1 @@
+= link_to('Back to List of Tokens', tokens_path, title: 'Back to List of Tokens', class: 'btn btn-outline-secondary px-4 mt-3 mt-sm-0')
```

Comment:
💭  Browsers have back buttons, is this really needed?

Reply:
In this case I would say yes. This 'Back to List of Tokens' link will be shown after a token has been created. If you press the back button of your browser, you will not request a page in the server, but you will still see the previous list of tokens, without the newly created one. It doesn't refresh automatically.

Do you have some other solution in mind?